shared/constants/form.ts

```diff
@@ -91,6 +91,9 @@ export const PAYMENT_VARIABLE_INPUT_AMOUNT_FIELD_ID =
 export const CLIENT_RADIO_OTHERS_INPUT_VALUE =
   '!!FORMSG_INTERNAL_RADIO_OTHERS_VALUE!!'
 
+export const CLIENT_CHECKBOX_OTHERS_INPUT_VALUE =
```

Comment:
Didn't see this being used by FE. Is it used somewhere? Saw this value is `CHECKED`, but not `SET`.

Reply:
In the FE, it is called `CHECKBOX_OTHERS_INPUT_VALUE` which is appended to checkbox value array when others option is checked. 

Note that the `othersInput` is always sent to the BE regardless of whether the others option is checked as long as the others input text box is filled. Not ideal behavior imo but not really affecting the functionality of the app, hence i am not aiming to fix this in this PR to avoid scope creep. 

Additional context: This behavior where `othersInput` is always sent to the BE also appears for radio button, which caused the bug [invalid submission responses for radio button when others input filled](https://linear.app/ogp/issue/FRM-1868/mrf-radio-bug-or-invalid-submission-responses-collected-for-radio) which has been hotfixed as it was causing responses to be submitted incorrectly.

Reply:
Thoughts on connecting these two values together?